package.json

```
@@ -0,0 +1,56 @@
+{
+  "name": "vite-react-typescript-starter",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "format:fix": "prettier --write \"src/**/*.{js,jsx,ts,tsx,json,css,scss,html}\"",
+    "preview": "vite preview",
+    "prepare": "husky",
+    "deploy": "gh-pages -d dist",
+    "predeploy": "npm run build",
+    "test": "vitest run",
+    "test-coverage": "vitest --coverage",
```

```suggestion
    "test:coverage": "vitest --coverage",
```
Мелочь, но тут тоже лучше найминг сохранять